src/quo/components/share/share_qr_code/style.cljs

```diff
@@ -1,45 +1,120 @@
 (ns quo.components.share.share-qr-code.style
-  (:require
-    [quo.foundations.colors :as colors]))
-
-(def qr-code-container
-  {:padding-top        12
-   :padding-horizontal 12
-   :padding-bottom     8
-   :border-radius      16
-   :background-color   colors/white-opa-5
-   :flex-direction     :column
-   :justify-content    :center
-   :align-items        :center})
-
-(def profile-address-column
-  {:margin-horizontal :auto
-   :flex              4})
-
-(def profile-address-container
-  {:flex-direction  :row
-   :justify-content :space-between
-   :margin-top      6})
+  (:require [quo.foundations.colors :as colors]))
+
+(def outer-container
+  {:border-radius 16
+   :width         "100%"
+   :overflow      :hidden})
+
+(def overlay-color colors/white-opa-5)
+
+(def ^:private padding-horizontal 12)
+
+(def content-container
+  {:z-index            1
+   :padding-horizontal padding-horizontal
+   :padding-top        12
+   :padding-bottom     8})
+
+;; Header
+(def header-container
+  {:flex-direction :row
+   :margin-bottom  12})
+
+(def header-tab-active {:background-color colors/white-opa-20})
+(def header-tab-inactive {:background-color colors/white-opa-5})
+(def space-between-tabs {:width 8})
+
+(def info-icon
+  {:margin-left :auto
+   :align-self  :center})
+
+(def info-icon-color colors/white-opa-40)
+
+;; QR code
+(defn qr-code-size
+  [total-width]
+  (- total-width (* 2 padding-horizontal)))
+
+(defn qr-image-background-color
+  [customization-color]
+  (colors/resolve-color customization-color :light))
```

Comment:
Should this be hardcoded?

Reply:
Yep!
The QR code component is displaying the backgrounds of wallet avatars in light intensity in figma even when all the other components are displaying in dark:
![image](https://github.com/status-im/status-mobile/assets/90291778/a0b2c1d6-ae52-418e-8969-35f9798bfab4)

https://www.figma.com/file/WQZcp6S0EnzxdTL4taoKDv/Design-System-for-Mobile?type=design&node-id=1189-28344&mode=design&t=dmJJUCTv3rXg0AqP-4

Reply:
Weird - did design confirm this?

Reply:
Oh actually if this is just network colors, they are always in 'light' theme - so this should resolve the same if :dark is passed here 🤔

Reply:
Which in that case better to pass theme param here for future proofing 👍

Reply:
oh no, in this case, the background of the avatar component is a customization-color, not a network color. (network colors are used to show `:eth:opt:arb1:...` in the `:wallet-multichain` variant of this QR code

Reply:
Got it, thanks

Reply:
> Weird - did design confirm this?

IMO, is clear because this component is not having light and dark variants, so it means it'll show the same regardless the theme. and the wallet avatar background is in light variant so it means it's always like that.

I'll ask designers to confirm this :+1:

Reply:
Yeah I more so mean that it seems like an error in the designs

Reply:
Also something to consider, perhaps here it is actually better to pass the unresolved customization-color here and instead wrap the avatar component used within this component in a theme provider with light mode. That leaves the colors responsibility of that component to the avatar itself should something change there.

Reply:
> Also something to consider, perhaps here it is actually better to pass the unresolved customization-color here and instead wrap the avatar component used within this component in a theme provider with light mode. That leaves the colors responsibility of that component to the avatar itself should something change there.

Yep, makes totally sense!
I'll add it, also, designers confirmed that the designs are correct, so the avatar will be always in light mode :+1: